recipes/soci/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.1)
+project(conanSociTest CXX)
+
+set(TEST_NAME soci.tests)
+add_executable(${TEST_NAME} test_package.cpp)
+target_compile_features(${TEST_NAME} PRIVATE cxx_std_11)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+conan_target_link_libraries(${TEST_NAME} ${CONAN_LIBS})
```

use the boilerpate please

```suggestion
cmake_minimum_required(VERSION 3.1)
project(test_package CXX)

include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
conan_basic_setup(TARGETS)

add_executable(${PROJECT_NAME} test_package.cpp)
target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
```